src/generate-erc721-permit-signature.ts

```diff
@@ -0,0 +1,167 @@
+import { getPayoutConfigByNetworkId } from "./utils/payoutConfigByNetworkId";
+import { BigNumber, ethers, utils } from "ethers";
+import { MaxUint256 } from "@uniswap/permit2-sdk";
+import { keccak256, toUtf8Bytes } from "ethers/lib/utils";
+
+const NFT_MINTER_PRIVATE_KEY = process.env.NFT_MINTER_PRIVATE_KEY ?? "0x59c6995e998f97a5a0044966f0945389dc9e86dae88c7a8412f4603b6b78690d";
+const NFT_CONTRACT_ADDRESS = "0x38a70c040ca5f5439ad52d0e821063b0ec0b52b6"; // "0x6a87f05a74AB2EC25D1Eea0a3Cd24C3A2eCfF3E0";
```

Comment:
What are these default values?

Reply:
the anvil instance of the deployed contract and anvil private key, I'll switch to env